<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoOrder</class>
 <widget class="QWidget" name="InfoOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="accept_pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>401</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Клиент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Номер телефона</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Адрес погрузки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Адрес доставки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Вес груза</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Описание</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="client_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="email_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="numberPhone_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="pointPogr_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="point_Dostav_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="vesGruza_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="description_lineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
